### PR TITLE
ISPN-9588 Workaround for no JGroups view after coordinator leaving

### PR DESCRIPTION
**Pull request by danberindei**
https://issues.jboss.org/browse/ISPN-9588

This is just a workaround, please don't resolve the issue.
Since I've been exercising the long tests watcher I've made some small improvements in that area as well.

### CODE REVIEW

File: commons-test/src/main/java/org/infinispan/commons/test/TestSuiteProgress.java [hunk 1]
```diff
@@ -96,7 +96,7 @@ synchronized void progress(String color, CharSequence message, Throwable t) {
          actualColor = color;
          actualReset = RESET;
       }
-      out.printf("%s[OK: %5s, KO: %5s, SKIP: %5s] %s%s%n", actualColor, succeeded.get(), failed.get(), skipped.get(), message, actualReset);
+      out.printf("%s[OK:%5s, KO:%3s, SKIP:%3s] %s%s%n", actualColor, succeeded.get(), failed.get(), skipped.get(), message, actualReset);
```
Comment (tristantarrant) [inline commons-test/src/main/java/org/infinispan/commons/test/TestSuiteProgress.java]: I guess if we have more than 999 failures, who cares about alignment :)

Comment (alanfx) [reply commons-test/src/main/java/org/infinispan/commons/test/TestSuiteProgress.java]: I've got 999 problems, but alignment ain't one!

Comment (danberindei) [reply commons-test/src/main/java/org/infinispan/commons/test/TestSuiteProgress.java]: I hope it doesn't mess with anyone's scripts, but I have one that adds an escape sequence to go up one line after a test progress line, so the more progress messages fit into a terminal line, the less scrolling.

**Comment by danberindei** [comment]
Unfortunately [Build #3](https://ci.infinispan.org/job/Infinispan/job/PR-6329/3) still has some hanged tests.

**Comment by danberindei** [comment]
The next version of JGroups will have a fix, closing.

**Pull request closed without merging**